src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import { Menu, X } from "react-feather";
+import MenuItems from "./MenuItems";
+import "./Navbar.css";
```

Comment:
Is style file for this component actually not called Navbar.scss? 

Cant spot anything else just now - looks good :)

Reply:
Done :)